MzmlReader/MzmlReader.cs

```diff
@@ -85,7 +87,12 @@ public Run LoadMzml(string path, double massTolerance, bool storeScansInMemory,
         public void ReadSourceFileMetaData(XmlReader reader, Run run)
         {
             bool cvParamsRead = false;
-            run.SourceFileName = reader.GetAttribute("name");
+            run.SourceFileName = Path.GetFileNameWithoutExtension(reader.GetAttribute("name"));
+            if (run.SourceFileName.ToLower().EndsWith(".wiff"))//in a .wiff.scan file you need to remove both extensions

```

Comment:
Why are we doing this? I don't understand

Reply:
We had two variables, run.SourceFileName and run.SourceFileType. Previously we weren't using type, we simply had the full filename of the file from which the mzML was made with extension under run.SourceFileName, for example "myFile.wiff.scan". At the end when writing files, we are now incorporating the name of the original file into the name of the file we are writing so as to distinguish from the other files when running multiple files for example "metrics_myFile.json". So we can remove the filetype from the name at the point of writing the file, which was my original plan. But then I noticed we still had the run.SourceFileType variable that we weren't using so I decided to remove the filetype from the name at the beginning and put it into the variable run.SourceFileType as we do have other sections where we use the filetype.